src/View.js

```diff
@@ -9,7 +9,18 @@ import UXBase from './UXBase';
  * @see: https://github.com/meetup/ux-capture#glossary
  */
 export default class View extends UXBase {
-	expectedZones = this.props.zoneConfigs.map(zoneConfig => {
+	expectedZones = this.setZones(this.props.zoneConfigs);
+
+	update(zoneConfigs) {
+		// Append new zones to existing config
```

Comment:
could you go ahead and add a 'TODO' comment here about needing to determine what to do about adding marks to the existing `expectedZones` rather than appending to `expectedZones` - it sounds like that will be the more common use-case

Reply:
`update` only gets called by `UXCapture.updateView()` which no one is using at the moment. I'll add the todo and we can update the inner workings.

@sergeychernyshev issues on the upstream repo are hard to track. Are those added as jira tickets/stories?